### ISSUE TITLE
LoginCollectNotice2

### ISSUE THREAD

**Issue by BobbyDuggart** [Dustio/Gunkolle#19]
No idea why, but the login message skipper isn't working right now. I think it has something to do with the Costume Lucky Bag thingy. On LoginCollectNotice2 it went to the shop instead of going to the home screen. Not a big deal, it'll probably fix itself when the anniversary event ends but I just wanted to point it out.

**Comment by Dustio** [comment]
Yeah I noticed that too, I haven't been able to see why it's clicking the event to go to the shop yet. The login collect is a pain because you only have one shot per day with little testing.
